recipes/docs_versions_menu/meta.yaml

```
@@ -0,0 +1,52 @@
+{% set name = "docs-versions-menu" %}
+{% set version = "0.5.0rc1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/docs_versions_menu-{{ version }}.tar.gz
+  sha256: 404d720f28cac80791244021713807a2593ad94c14116acf6247eed788624091
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  entry_points:
+    - docs-versions-menu = docs_versions_menu.cli:main
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+    - setuptools
+
+  run:
+    - python >=3.6
+    - jinja2
+    - click >=6.7
+    - packaging
+    - pyparsing >=2.0.2
+    - sphinx
+
+test:
+  imports:
+    - docs_versions_menu
+
+  commands:
+    - docs-versions-menu --help
```

Please add a check that all Python requirements are fulfilled.
```suggestion
  requires:
    - pip
  commands:
    - docs-versions-menu --help
    - pip check
```

---

Sure!